README.md

```diff
@@ -693,9 +693,13 @@ Other options:
  - `toplevel` — mangle names declared in the toplevel scope (disabled by
   default).
 
-  - `eval` — mangle names visible in scopes where eval or with are used
+   - `eval` — mangle names visible in scopes where eval or with are used
   (disabled by default).
 
+ - `keep_fnames` -- default `false`.  Pass `true` to not mangle
```

Comment:
Oh, I misunderstood the problem..
So, the `eval` line must not be nested, right?